packages/expo-cli/src/commands/info/upgrade.ts

```diff
@@ -10,7 +10,8 @@ export default function (program: Command) {
       .description('Upgrade the project packages and config for the given SDK version')
       .helpGroup('info')
       .option('--npm', 'Use npm to install dependencies. (default when package-lock.json exists)')
-      .option('--yarn', 'Use Yarn to install dependencies. (default when yarn.lock exists)'),
+      .option('--yarn', 'Use Yarn to install dependencies. (default when yarn.lock exists)')
+      .option('--pnpm', 'Use pnpm to install dependencies. (default when pnpm-lock.yaml exists)'),
```

Comment:
The package manager change is fine but we should be adding new features to the versioned CLI so we don't have to wrangle issues across tools. The expo-cli package should not have new features added to it.

Reply:
Reverted!